src/components/CohortBuilder/Summary/index.js

```diff
@@ -41,10 +50,24 @@ const Summary = () => (
         </Col>
         <Col sm={12} md={md} lg={lg}>
           <CardSlot title="Most Frequent Diagnoses">
+            <HorizontalBar style={{maxWidth: '100px'}}
+              data={topDiagnosesBarMock}
+              indexBy="label"
+              keys={['probands', 'familyMembers']}
+              tooltipFormatter={participantTooltip}
+              sortByValue={true}
+              tickInterval={4}
+              colors={[theme.chartColors.blue, theme.chartColors.purple]}
```

Comment:
the blue color is rendering in black for me in the [preview here](https://5c51e240bc31f5000712d1b2--kf-portal-storybook.netlify.com/?selectedKind=CohortBuilder&selectedStory=Page%20Layout&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel). Not sure if it's just me but probably worth investigating.
![screen shot 2019-01-30 at 1 03 21 pm](https://user-images.githubusercontent.com/11821106/52002698-a3974080-2490-11e9-9a5b-40c3c4879242.png)

Reply:
was uncommited, commited